tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -606,4 +606,91 @@ class CompletionCaseSuite extends BaseCompletionSuite {
     "",
   )
 
+  check(
+    "exhaustive-enum-tags".tag(IgnoreScala2),
+    s"""|object Tags:
+        |  trait Hobby
+        |  trait Chore
+        |  trait Physical
+        |
+        |
+        |import Tags.*
+        |
+        |enum Activity:
+        |  case Reading(book: String, author: String) extends Activity, Hobby
+        |  case Sports(time: Long, intensity: Double) extends Activity, Physical, Hobby
+        |  case Cleaning                              extends Activity, Physical, Chore
+        |  case Singing(song: String)                 extends Activity, Hobby
+        |  case DishWashing(amount: Int)              extends Activity, Chore
+        |
+        |import Activity.*
+        |
+        |def energySpend(act: Activity & (Physical | Chore)): Double = 
```

Comment:
Actually, maybe we should test another scenario with `Hobby | Physical & Chore`, which should show use `Reading`, `Singing` and `Cleaning`

Reply:
Shouldn't it show `Reading`, `Singing`, `Cleaning` and `Sports`?